integration_tests/setup_db.py

```diff
@@ -362,6 +357,7 @@ def _setup_event_data(
             date=data["date"],
             event_type_name=data["event_type_value"],
             family_import_id=data["family_import_id"],
+            # family_document_import_id=data["family_document_import_id"],
```

Comment:
Would be nice to have a test that checked this is set

Reply:
Have readded the todo that seemed to get lost during the merge.